libbpf-tools/oomkill.c

```diff
@@ -16,6 +16,7 @@
 #include <bpf/libbpf.h>
 #include "oomkill.skel.h"
 #include "oomkill.h"
+#include "min_core_btf_helpers.h"
```

Comment:
sorry, another naming nit: I'd generalize this header to be just "btf_helpers.h" so that we can use it for any other BTF-related helpers in the future (e.g., if we need some BTF-based feature probing helpers or something like that). "min_core_btf_helpers.h" is a bit too specific.